numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -508,6 +551,8 @@ def _get_include_dir():
     """Find the root include directory."""
     options = [
         ("Conda environment (NVIDIA package)", get_conda_include_dir()),
+        ("NVIDIA NVCC Wheel", get_wheel_include()),
+        ("NVIDIA CCCL Wheel", get_cccl_wheel_include()),
```

Comment:
The `_find_valid_path` function below basically takes the first non None result from this `options` list. We need `get_cccl_wheel_include` to by included into `get_wheel_include` as opposed to a separate option here.

Reply:
This is the part I was just starting to get familiar with, I had hoped it would use all of these, not just the first one. That makes sense.